hydromt_wflow/wflow.py

```diff
@@ -4649,48 +4666,56 @@ def setup_irrigation(
             }
```

Comment:
I cant comment but I also made a mistake in the line up, should be last letter:
k: v for k, v in output_names.items() if "irrigated-paddy" in **k**